xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -453,9 +464,18 @@ void CSelectionStreams::Update(const std::shared_ptr<CDVDInputStream>& input,
       AudioStreamInfo info = nav->GetAudioStreamInfo(i);
       s.name     = info.name;
       s.codec    = info.codecName;
-      s.codecDesc = info.codecDesc;
+      if (aStream)
+      {
+        s.codecDesc = aStream->GetStreamType();
+        s.channels = aStream->iChannels;
+        s.bitrate = aStream->iBitRate;
+      }
+      else
+      {
+        s.codecDesc = info.codecDesc;
+        s.channels = info.channels;
+      }
```

Comment:
if you want keep this mixage you have to explain the reason because to me its unclear,
im running the original code without your "aStream" condition and i can get the codecDesc/channels correctly also on menu interludes

Reply:
It's not about the codecDesc in menus.
The reason is codec + channel count that seems more accurate from ffmpeg than in ifo: there is no value defined for DTS-ES in ifo, and I have a DVD where channel count in ifo is not accurate. Then, might as well use the opportunity to grab the bitrate,,not available in ifo. That one makes a small difference in the stream selection sort order.
I don't care that much about the commit, I'm not much of a DVD user. For the most part I just have a couple handy for regression testing.